sources/packages/web/src/components/common/students/StudentRestrictions.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header title="All restrictions">
+        <template #actions v-if="canAddRestrictions">
+          <check-permission-role :role="Role.StudentAddRestriction">
+            <template #="{ notAllowed }">
+              <v-btn
+                @click="addStudentRestriction"
+                class="float-right"
+                color="primary"
+                data-cy="addRestrictionButton"
+                prepend-icon="fa:fa fa-plus-circle"
+                :disabled="notAllowed"
+                >Add restriction</v-btn
+              >
+            </template>
+          </check-permission-role>
+        </template>
+      </body-header>
+    </template>
+    <content-group>
+      <DataTable
+        :value="studentRestrictions"
+        :paginator="true"
+        :rows="DEFAULT_PAGE_LIMIT"
+        :rowsPerPageOptions="PAGINATION_LIST"
+      >
+        <template #empty>
```

Comment:
We no longer use `template #empty`, can you please change it to use the `toggle-content`?